deploy/crds/argoproj.io_argocds_crd.yaml

```diff
@@ -164,844 +164,854 @@ spec:
                     annotations:
                       additionalProperties:
                         type: string
-                      description: Annotations is the map of annotations to apply
-                        to the Ingress.
-                      type: object
-                    enabled:
-                      description: Enabled will toggle the creation of the Ingress.
-                      type: boolean
-                    path:
-                      description: Path used for the Ingress resource.
-                      type: string
-                    tls:
-                      description: TLS configuration. Currently the Ingress only supports
-                        a single TLS port, 443. If multiple members of this list specify
-                        different hosts, they will be multiplexed on the same port
-                        according to the hostname specified through the SNI TLS extension,
-                        if the ingress controller fulfilling the ingress supports
-                        SNI.
-                      items:
-                        description: IngressTLS describes the transport layer security
-                          associated with an Ingress.
-                        properties:
-                          hosts:
-                            description: Hosts are a list of hosts included in the
-                              TLS certificate. The values in this list must match
-                              the name/s used in the tlsSecret. Defaults to the wildcard
-                              host setting for the loadbalancer controller fulfilling
-                              this Ingress, if left unspecified.
-                            items:
+                      tls:
```

Comment:
The new changes looks to have broken the indentation in many places. Can we fix them ?

error: error validating "deploy/crds/argoproj.io_argocds_crd.yaml": error validating data: [ValidationError(CustomResourceDefinition.spec.versions[0].schema.openAPIV3Schema.properties.spec.properties.grafana.properties.ingress.properties.type): invalid type for io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.JSONSchemaProps: got "string", expected "map", ValidationError(CustomResourceDefinition.spec.versions[0].schema.openAPIV3Schema.properties.spec.properties.grafana.properties.ingress): unknown field "resources" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.JSONSchemaProps, ValidationError(CustomResourceDefinition.spec.versions[0].schema.openAPIV3Schema.properties.spec.properties.grafana.properties.ingress): unknown field "route" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.JSONSchemaProps, ValidationError(CustomResourceDefinition.spec.versions[0].schema.openAPIV3Schema.properties.spec.properties.grafana.properties.ingress): unknown field "size" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.JSONSchemaProps, ValidationError(CustomResourceDefinition.spec.versions[0].schema.openAPIV3Schema.properties.spec.properties.grafana.properties.ingress): unknown field "version" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.JSONSchemaProps, ValidationError(CustomResourceDefinition.spec.versions[0].schema.openAPIV3Schema.properties.spec.properties.grafana.properties.required): invalid type for io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.JSONSchemaProps: got "array", expected "map", ValidationError(CustomResourceDefinition.spec.versions[0].schema.openAPIV3Schema.properties.spec.properties.grafana.properties.type): invalid type for io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.JSONSchemaProps: got "string", expected "map"]; if you choose to ignore these errors, turn validation off with --validate=false